docker/entrypoint-generate.sh

```diff
@@ -9,7 +9,7 @@ chown -R keybase:keybase /mnt
 sudo -i -u keybase bash << EOF
 source docker/env.sh
 export "FORCE_WRITE=$FORCE_WRITE"
-nohup bash -c "run_keybase -g &"
+nohup bash -c "run_keybase -g 2>&1 | grep -v 'KBFS failed to FUSE mount' &"
```

Comment:
i thought there was an option to run_keybase that prevents this from trying to mount. `-f` maybe?

Reply:
Yup, `-f` exists (docs: `-f  do not start KBFS (env KEYBASE_NO_KBFS=1)`) but the issue is that then `keybase fs ...` commands will not work. What we want is KBFS but without FUSE which isn't an existing option. 

If you run it with `-f` the error message from `keybase fs ...` is: 

```
[ERRO keybase main.go:86] 001 Keybase services aren't running - KBFS client not found. On Linux you need to start them after an update with `run_keybase` command.
```

So I think it makes sense to just silence the FUSE error and keep running.